spec/rubocop/cop/layout/space_before_block_braces_spec.rb

```diff
@@ -71,4 +71,55 @@
       expect_no_offenses('each{ puts }')
     end
   end
+
+  context 'with space before empty braces not allowed' do
```

Comment:
We can put this context inside the `'EnforcedStyle' => 'space'` one, for example.

We can event extract these example into a shared examples group and include it into `'EnforcedStyle' => 'space'` (without explicit `'EnforcedStyleForEmptyBraces'`) and this context (with explicit `'EnforcedStyleForEmptyBraces'`).